client/src/components/PatientSearchBar/PatientSearchBar.tsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from 'react';
+import { useGetPatientsByNameLazyQuery } from '../../generated/graphql-types';
+import SearchBar from '../SearchBar/SearchBar';
+import PatientSearchBarProps from './PatientSearchBar.type';
+
+export default function PatientSearchBar({
+  handlePatientSelected
+}: PatientSearchBarProps) {
+  const [search, setSearch] = useState<string>('');
+  const [getPatientsByname, { data }] = useGetPatientsByNameLazyQuery();
+
+  const handleChange = (value: string): void => {
+    setSearch(value);
+  };
+
+  useEffect(() => {
+    if (search !== '') {
```

Comment:
No need to not-equals on an empty string? Just if (search) no?

Reply:
for readibility